### PR TITLE
Italian translation for two user guides

### PR DESCRIPTION
**Pull request by qwert95**

### CODE REVIEW

**Comment by el00ruobuob** [comment]
Looks ok at first sight. I'll test it later.

File: _i18n/it/resources/user-guides/importing_blockchain.md [hunk 1]
```diff
@@ -1,57 +1,55 @@
...
-### Step 2
+### Passo 2
 
-Find the path of your Monero wallet (the folder where you extracted your wallet). For example mine is:
+Trova il percorso del tuo wallet Monero (la cartella nella quale hai estratto il tuo wallet). Per esempio il mio è :
 
 `D:\monero-gui-0.10.3.1`
 
-Your path may be different depending on where you decided to download your wallet and what version of the Monero wallet you have.
+Il tuo percorso potrebbe essere diverso in base a dove hai deciso di scaricare il tuo wallet e alla versione del wallet Monero che stai utilizzando.
```
Comment (erciccione) [inline _i18n/it/resources/user-guides/importing_blockchain.md]: `Wallet` can be translated into `portafoglio`

File: _i18n/it/resources/user-guides/importing_blockchain.md [hunk 2]
```diff
@@ -1,57 +1,55 @@
...
-### Step 3
+### Passo 3
 
-Find the path of your downloaded Blockchain for example mine was: 
+Trova il percorso del file di bootstrap della Blockchain che hai scaricato, per esempio il mio è :
 
 `C:\Users\KeeJef\Downloads\blockchain.raw`
 
-Yours might be different depending on where you downloaded the Blockchain to.
+Il tuo potrebbe essere diverso in base a dove hai deciso di salvare il file di boostrap.
```
Comment (erciccione) [inline _i18n/it/resources/user-guides/importing_blockchain.md]: `salvare il file di boostrap` -> `salvare la blockchain`. For consistency

File: _i18n/it/resources/user-guides/importing_blockchain.md [hunk 3]
```diff
@@ -1,57 +1,55 @@
...
 
-### Step 4
+### Passo 4
 
-Open a Command Prompt window. You can do this by pressing the Windows key + R, and then typing in the popup box `CMD`
+Apri la finestra del Prompt dei comandi. Per farlo premi il tasto Windows + R, e poi scrivi nel campo `CMD` e premi Ok
```
Comment (erciccione) [inline _i18n/it/resources/user-guides/importing_blockchain.md]: `nel campo` doesn't sound right. Maybe `nella finestra` ?

File: _i18n/it/resources/user-guides/importing_blockchain.md [hunk 4]
```diff
@@ -1,57 +1,55 @@
...
-`cd C:\YOUR\MONERO\WALLET\FILE\PATH\HERE` 
+`cd C:\IL\TUO\PERCORSO\DEL\WALLET\MONERO\QUI` 
 
-It should look something like:
+Nel mio caso :
 
 `cd D:\monero-gui-0.10.3.1`
 
-If your Monero wallet is on another drive you can use `DriveLetter:` for example if your Monero wallet was on your D drive then before using the cd command you would do `D:`
+Se il tuo wallet Monero è in un'altra partizione puoi cambiarla digitando il comando `LetteraPartizione:`, per esempio se il tuo wallet è nella partizione C dovrai digitare `C:`
```
Comment (erciccione) [inline _i18n/it/resources/user-guides/importing_blockchain.md]: `wallet`  -> `portafoglio`
i would add a `,` after `è in un'altra partizione`

File: _i18n/it/resources/user-guides/importing_blockchain.md [hunk 5]
```diff
@@ -1,57 +1,55 @@
...
 `monero-blockchain-import --verify 1 --input-file C:\Users\KeeJef\Downloads\blockchain.raw`
 
-If you downloaded the Blockchain from a trusted, reputable source you may set `verify 0` this will reduce the amount of time to sync the Blockchain.  
+Se hai scaricato la Blockchain da una fonte sicura e affidabile potresti impostare `verify 0` per ridurre il tempo necessario a sincronizzare la Blockchain.
 
-### Step 7
+### Passo 7
 
-After the the Blockchain has finished syncing up you can open your Monero wallet normally. Your downloaded blockchain.raw can be deleted. 
+Dopo che la Blockchain sarà sincronizzata potrai aprire il tuo wallet Monero normalmente. Il file blockchain.raw può essere eliminato.
```
Comment (erciccione) [inline _i18n/it/resources/user-guides/importing_blockchain.md]: `wallet`->`portafoglio`

**Comment by qwert95** [comment]
@erciccione I'll for sure make the changes for better consistency, but I'm new to the git system.
Am I supposed to close this pull request, make the changes and open another one?

**Comment by el00ruobuob** [comment]
Not. Just make your changes commit and push

**Comment by erciccione** [comment]
as @el00ruobuob said, you can just edit directl on this pull request and push your changes

**Comment by erciccione** [comment]
Perfect. Now would be great to have those commits squashed. If you don't know how to do it i can help you with that. If you don't know git much and prefer to don't deal with it i can do it for you

**Comment by qwert95** [comment]
@erciccione yeah, thanks, looks like the best option for everyone at the moment :+1:

**Comment by erciccione** [comment]
No problem. Just add me as collaborator to your repo, so i can squash for you. You can find that option in the 'setting' page of your repository

**Pull request merged**